messages/en.json

```
@@ -30,5 +30,6 @@
   "admin/tiktok.disconnect.confirm.button": "Confirm",
   "admin/tiktok.disconnect.confirmation-toast.success": "Account disconnected successfully.",
   "admin/tiktok.disconnect.confirmation-toast.failure": "It was not possible to disconnect the account.",
-  "admin/tiktok.confirmation-modal.title": "Check the information before submitting"
+  "admin/tiktok.confirmation-modal.title": "Check the information before submitting",
+  "admin/tiktok.connection-error.access-token": "You was disconnected because of an AccessToken error."
```

```suggestion
  "admin/tiktok.connection-error.access-token": "You have been disconnected due to an access token error!"
```
I think it's better this way.